### ISSUE TITLE
An unhandled exception occurred. When there is space in path

### ISSUE THREAD

**Issue by aguacongas** [dotnet/try#197]
To repoduce :
- Clone the repo
- Checkout the branch samples
- Rename the folder by *dotnet try*
- Lanch dotnet try with:
``` bash
C:\Projects\OpenSources\dotnet try\Samples>dotnet try
```
- Navigate to http://localhost:5000/Beginners/HelloWorld.md
- Click the run button

The request:
``` json
POST http://localhost:5000/workspace/run?hostOrigin=http%3A%2F%2Flocalhost%3A5000%2FBeginners%2FHelloWorld.md

{
	"runArgs": "--region intro --source-file .\\myapp\\Program.cs --project .\\myapp\\myapp.csproj",
	"workspace": {
		"workspaceType": "C:\\Projects\\OpenSources\\dotnet try\\Samples\\Beginners\\myapp\\myapp.csproj",
		"files": [],
		"buffers": [
			{
				"id": "C:\\Projects\\OpenSources\\dotnet try\\Samples\\Beginners\\myapp\\Program.cs@intro",
				"content": "Console.WriteLine(\"Hello World!\");",
				"position": 0
			}
		]
	},
	"activeBufferId": "C:\\Projects\\OpenSources\\dotnet try\\Samples\\Beginners\\myapp\\Program.cs@intro",
	"requestId": "trydotnet.client_5"
}
```
returns the error:

``` json
Status: 500

{
	"message": "An unhandled exception occurred.",
	"exception": "System.NullReferenceException: Object reference not set to an instance of an object.\r\n   at WorkspaceServer.Servers.Roslyn.RoslynWorkspaceServer.EmitCompilationAsync(Compilation compilation, Package package) in E:\\A\\_work\\1811\\s\\WorkspaceServer\\Servers\\Roslyn\\RoslynWorkspaceServer.cs:line 287\r\n   at WorkspaceServer.Servers.Roslyn.RoslynWorkspaceServer.Run(WorkspaceRequest request, Budget budget) in E:\\A\\_work\\1811\\s\\WorkspaceServer\\Servers\\Roslyn\\RoslynWorkspaceServer.cs:line 253\r\n   at MLS.Agent.Controllers.RunController.Run(WorkspaceRequest request, String timeoutInMilliseconds) in E:\\A\\_work\\1811\\s\\MLS.Agent\\Controllers\\RunController.cs:line 73\r\n   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\r\n   at System.Threading.Tasks.ValueTask`1.get_Result()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync()"
}
```

If I rename *dotnet try* by *dotnet_try*, the code works.

In this .zip you'll find [.trydotnet-builderror + package_designTimeBuild.binlog](https://github.com/dotnet/try/files/3183091/myapp.zip)

**Comment by LadyNaggaga** [comment]
This is similar to issue [195]( https://github.com/dotnet/try/issues/195)

**Comment by jonsequitur** [comment]
This is fixed. A new package will be available shortly.

**Comment by aguacongas** [comment]
For the record, this is fixed by version 1.0.19266.1  
``` bash
> dotnet tool update --global dotnet-try
Tool 'dotnet-try' was successfully updated from version '1.0.19264.11' to version '1.0.19266.1'.
```
